Health/src/main/java/com/sciencesquad/health/health/Nutrition/NutritionModule.java

```diff
@@ -0,0 +1,81 @@
+package com.sciencesquad.health.health.Nutrition;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.sciencesquad.health.Module;
+
+import io.realm.RealmQuery;
+
+/**
+ * Created by danielmiller on 2/13/16.
+ *
+ * Nutrition Module itself.
+ *
+ * Note:
+ * It must be expanded upon from this current baby state
+ */
+public class NutritionModule extends Module {
+
+    private static final String TAG = "Nutrition Module";
+
+    private RealmNutritionModule nutritionRealm;
+
+    public NutritionModule(){
+        /**
+         * Constructs the module itself.
+         */
+        this.nutritionRealm = new RealmNutritionModule();
+        testNutritionModule();
+
+    }
+
+    protected void testNutritionModule(){
```

Comment:
We should think about adding an `androidTests` folder and dumping a ton of unit tests there right off the bat.